### PR TITLE
Add docs for package events

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

File: docs/advanced-features/package-events.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Package events
+
+Every time package revision is created in one of the Quilt stack buckets, Quilt stack
+emits event to default
```
Comment (sir-sigurd) [inline docs/advanced-features/package-events.md:4]: @akarve 
We were thinking that we need to create a bus in the stack for this, maybe we should do this before publishing docs?

File: docs/advanced-features/package-events.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        "version": "0.1",
+        "type": "created",
+        "bucket": "example",
+        "handle": "some/package",
+        "topHash": "a0fddace2eb2fd91faa697d237a5dbdcfa77f0fd38ca8b4c850dbd93d142ee69"
+    }
+}
+```
+
+To handle such events you need to create
```
Comment (drernie) [inline docs/advanced-features/package-events.md:28]:
```suggestion
To handle such events you need to create an
```

File: docs/advanced-features/package-events.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
+# Package events
+
+Every time package revision is created in one of the Quilt stack buckets, Quilt stack
```
Comment (drernie) [inline docs/advanced-features/package-events.md:3]:
```suggestion
Every time a package revision is created in one of the Quilt stack buckets, the Quilt stack
```

File: docs/advanced-features/package-events.md (hunk 1 above)
Comment (drernie) [inline docs/advanced-features/package-events.md:4]:
```suggestion
emits event to the default
```

File: docs/advanced-features/package-events.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+emits event to default
+[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html)
+bus. Event has the following structure:
```
Comment (drernie) [inline docs/advanced-features/package-events.md:6]:
```suggestion
bus. The event has the following structure:
```

**Review by drernie — DISMISSED** [review]

File: docs/advanced-features/package-events.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+# Package events
+
+Every time a package revision is created in one of the Quilt stack buckets, the
+Quilt stack emits event to the default
+[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html)
+bus. The event has the following structure:
```
Comment (akarve) [inline docs/advanced-features/package-events.md:6]: Each time a package is created or revised in a Quilt stack bucket,
the stack emits a `package-revision` event on the default
[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html) bus.
Events have the following structure:

Comment (akarve) [inline docs/advanced-features/package-events.md:6]:
```suggestion
When a package is created or revised in a Quilt stack bucket,
the stack emits a `package-revision` event on the default
[EventBridge](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html) bus.
These events have the following structure:
```

File: docs/advanced-features/package-events.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        "type": "created",
+        "bucket": "example",
+        "handle": "some/package",
+        "topHash": "a0fddace2eb2fd91faa697d237a5dbdcfa77f0fd38ca8b4c850dbd93d142ee69"
+    }
+}
+```
+
+To handle such events you need to create an
+[EventBridge rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rules.html).
```
Comment (akarve) [inline docs/advanced-features/package-events.md:29]:
```suggestion
You can create an
[EventBridge rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rules.html)
similar to the following to respond to `package-revision` events:
```

File: docs/advanced-features/package-events.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+  To customize what events are processed and how they are processed
+  you need to modify `EventBridgeRule`.
+
+  See https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns.html
+  for syntax of event patterns.
+
+  See https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-targets.html
+  for a list of supported targets.
+
```
Comment (akarve) [inline docs/advanced-features/package-events.md:52]:
```suggestion
  Demonstrate how to respond to package events.

  Modify `EventBridgeRule` to customize event processing. 

  See https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns.html
  for event pattern syntax.

  See https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-targets.html
  for rule targets.
```

File: docs/advanced-features/package-events.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+Parameters:
+  PackageBucket:
+    Type: String
+  PackageNamePrefix:
+    Type: String
+    Description: Leave empty to match every package.
+  EmailToSubscribe:
+    Type: String
+    Description: Email that will get package notifications.
```
Comment (akarve) [inline docs/advanced-features/package-events.md:61]:
```suggestion
    Description: Confirm subscription over email to receive a copy of package events
    that occur under `PackageNamePrefix` in `PackageBucket`.
```

**Review by akarve — COMMENTED** [review]
Thanks for putting this together.

**Review by drernie — APPROVED** [review]

**Pull request merged**